packages/vscode-extension/package.nls.json

```diff
@@ -45,57 +45,53 @@
     "teamstoolkit.codeFlowLogin.checkOnlineFailTitle": "CheckOnlineFail",
     "teamstoolkit.codeFlowLogin.checkOnlineFailDetail": "You appear to be offline. Please check your network connection.",
     "teamstoolkit.codeLens.projectSettingsNotice": "This file is maintained by Teams Toolkit, please do not modify it",
-    "teamstoolkit.commands.accounts.title": "Teams: Accounts",
+    "teamstoolkit.commands.accounts.title": "Accounts",
     "teamstoolkit.commands.accountsLink.title": "Learn more about accounts",
     "teamstoolkit.commands.addAppOwner.title": "Add M365 Teams App (with AAD App) Owners",
-    "teamstoolkit.commands.addCapability.title": "Teams: Add capabilities",
-    "teamstoolkit.commands.addCICDWorkflows.title": "Teams: Add CI/CD Workflows",
-    "teamstoolkit.commands.addCloudResource.title": "Teams: Add cloud resources",
-    "teamstoolkit.commands.addFeature.title": "Teams: Add features",
-    "teamstoolkit.commands.addSso.title": "Teams: Add SSO",
-    "teamstoolkit.commands.connectExistingApi.title": "Teams: Add connection to an API",
+    "teamstoolkit.commands.addFeature.title": "Add features",
+    "teamstoolkit.commands.azureAccountSettings.title": "Azure portal",
     "teamstoolkit.commands.createAccount.azure": "Create an Azure account",
     "teamstoolkit.commands.createAccount.free": "Free",
     "teamstoolkit.commands.createAccount.m365": "Create a M365 testing tenant",
     "teamstoolkit.commands.createAccount.title": "Create an account",
-    "teamstoolkit.commands.createEnvironment.title": "Teams: Create new environment",
-    "teamstoolkit.commands.createProject.title": "Teams: Create a new Teams app",
+    "teamstoolkit.commands.createEnvironment.title": "Create new environment",
```

Comment:
If we remove the 'Teams:' prefix, does that mean we'll need to update the svg in quick start?

Reply:
In command palette it will show "Teams: Create new environment" as before, so we don't need to update the SVG.